src/classes/ng-uploader-options.class.ts

```diff
@@ -22,44 +22,44 @@ export interface INgUploaderOptions {
 
 export class NgUploaderOptions implements INgUploaderOptions {
   url: string;
-  cors?: boolean;
-  withCredentials?: boolean;
-  multiple?: boolean;
+  cors: boolean;
+  withCredentials: boolean;
+  multiple: boolean;
   maxUploads?: number;
   data?: any;
-  autoUpload?: boolean;
+  autoUpload: boolean;
   multipart?: any;
-  method?: 'POST' | 'GET';
-  customHeaders?: any;
-  encodeHeaders?: boolean;
-  authTokenPrefix?: string;
+  method: 'POST' | 'GET';
+  customHeaders: any;
+  encodeHeaders: boolean;
+  authTokenPrefix: string;
   authToken?: string;
-  fieldName?: string;
-  fieldReset?: boolean;
+  fieldName: string;
+  fieldReset: boolean;
   previewUrl?: string;
-  calculateSpeed?: boolean;
-  filterExtensions?: boolean;
-  allowedExtensions?: string[];
+  calculateSpeed: boolean;
+  filterExtensions: boolean;
+  allowedExtensions: string[];
 
   constructor(obj: INgUploaderOptions) {
-    this.url = obj.url != null ? obj.url : '';
-    this.cors = obj.cors != null ? obj.cors : true;
-    this.withCredentials = obj.withCredentials != null ? obj.withCredentials : this.withCredentials;
-    this.multiple = obj.multiple != null ? obj.multiple : true;
-    this.maxUploads = obj.maxUploads != null ? obj.maxUploads : 10;
-    this.data = obj.data != null ? obj.data : {};
+    this.url = obj && obj.url ? obj.url : '';
```

Comment:
hey @krzysztof-miemiec. thank you for your PR. can you please update to 
```ts
this.url = obj && obj.url != null ? obj.url : ''; 
```
and also elsewhere. thanks.